components/tools/OmeroWeb/omeroweb/webgateway/util.py

```diff
@@ -34,6 +34,15 @@ def getIntOrDefault(request, name, default):
     return index
 
 
+def get_longs(request, name):
+
```

Comment:
Move the deleted webclient get_longs docstring here:
"""
Retrieves parameters from the request. If the parameters are not present
an empty list is returned
This does not catch exceptions as it makes sense to throw exceptions if
the arguments provided do not pass basic type validation
"""